### PR TITLE
feat(web3): Add web3.eth.accounts compatible provider

### PR DESCRIPTION
**Pull request by RetricSu**

### CODE REVIEW

File: packages/ethers/README.md [hunk 1]
```diff
@@ -1,37 +1,34 @@
...
+yarn add @polyjuice-provider/ethers 
 ```
 
-with
+or
 
-```ts
-import { PolyjuiceWallet } from "@polyjuice-provider/ethers";
+```sh
+npm install @polyjuice-provider/ethers
```
Comment (classicalliu) [inline packages/ethers/README.md:14]:
```suggestion
npm install --save @polyjuice-provider/ethers
```

File: packages/web3/README.md [hunk 2]
```diff
@@ -1,48 +1,74 @@
...
 
 ```sh
 yarn add @polyjuice-provider/web3 
 ```
 
-## Deploy contract
+or
+
+```sh
+npm install @polyjuice-provider/web3
```
Comment (classicalliu) [inline packages/web3/README.md:14]:
```suggestion
npm install --save @polyjuice-provider/web3 
```

File: packages/web3/src/accounts.ts [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+}
+
+export function TxConfigValueTypeToString(value: number | string | BN) {
+  if (typeof value === "string") {
+    value = "0x" + BigInt(value).toString(16);
+  }
+  if (typeof value === "number") {
+    value = "0x" + BigInt(value).toString(16);
+  }
+  if (typeof value != "string" || typeof value != "number") {
```
Comment (classicalliu) [inline packages/web3/src/accounts.ts:149]:
```suggestion
  if (typeof value !== "string" || typeof value !== "number") {
```

File: packages/web3/src/accounts.ts [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+    // BN.js type
+    value = value.toString(16);
+  }
+  return value;
+}
+
+// todo: move to @polyjuice-provider/godwoken
+export function calcPolyjuiceTxHash(tx: RawL2Transaction){
+  const tx_hash = lumosUtils.ckbHash(SerializeRawL2Transaction(normalizer.NormalizeRawL2Transaction(tx))).serializeJson().slice(0, 66);
```
Comment (classicalliu) [inline packages/web3/src/accounts.ts:158]: why `slice` here ?

Comment (RetricSu) [reply packages/web3/src/accounts.ts:158]: oh, did not realize it is already 32 bytes...

Comment (RetricSu) [reply packages/web3/src/accounts.ts:158]:
```suggestion
  const tx_hash = lumosUtils.ckbHash(SerializeRawL2Transaction(normalizer.NormalizeRawL2Transaction(tx))).serializeJson();
```

**Pull request merged**